### DISCUSSION TITLE
Hubs on Google Cardboard?

### DISCUSSION THREAD

**Discussion by derek-jackson** [mozilla/hubs#5973] [General Questions & Discussion]
Hi, I've created a custom scene and am trying to view it using Google Cardboard and the Chrome browser on my iPhone.

I've followed the link and entered the code, but it just sits there spinning on the Loading Scene screen until the session times out. The scene loads fine in the standard web viewer - what might be the issue with viewing it using Cardboard?

**Comment by pattersonbl2** [comment]
This is more than likely the case.  I am unfamiliar with google cardboard so I would need to look a little more into this.
